cocos/2d/CCLabel.cpp

```diff
@@ -42,6 +43,102 @@ NS_CC_BEGIN
 
 const int Label::DistanceFieldFontSize = 50;
 
+class LabelLetter : public Sprite
+{
+#if CC_SPRITEBATCHNODE_RENDER_SUBPIXEL
```

Comment:
This macro is defined in `CCSprite.cpp`, we shouldn't define it again just for DRY principle.
You should move it to CCSprite.h.